README.md

```diff
@@ -4,20 +4,29 @@ You can deploy 1Password SCIM Bridge on any supported infrastructure that allows
 
 ## Automatic (one-click) deployment
 
-The easiest way to deploy 1Password SCIM Bridge is with our one-click apps, which are currently available for Google Cloud Platform and DigitalOcean:
+For a guided deployment, you can use our Google Cloud or DigitalOcean marketplace app to deploy 1Password SCIM Bridge:
```

Comment:
As I read this, I wondered what "guided" means, since we have guides (tutorials) for both methods. We should imply that it's easier, which is more accurate:

```suggestion
If you'd prefer to deploy 1Password SCIM Bridge without the command line, you can use the Google Cloud Platform or DigitalOcean marketplace app:
```

I'm open to other suggestions here of course, but I do think that some readers may feel in over their head with all the commands.

Reply:
I like this phrasing! Slight adjustments to remove "the" and add a comma:

```suggestion
If you'd prefer to deploy 1Password SCIM Bridge using a marketplace, you can use Google Cloud Platform or DigitalOcean:
```